### PR TITLE
ACL examples

### PR DESCRIPTION
**Pull request by jerjou**
* Example for getting ACL of bucket for specific user
* Example for getting ACL of blob for specific user

### CODE REVIEW

**Comment by jerjou** [comment]
@mziccard FYI

**Comment by garrettjonesgoogle** [comment]
@jerjou , you will want to cc me from this point forward.

File: google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java [hunk 1]
```diff
@@ -718,6 +718,19 @@ public Acl getBucketAcl(String bucketName) {
   }
 
   /**
+   * Example of getting the ACL entry for a specific user on a bucket.
+   */
+  // [TARGET getAcl(String, Entity)]
+  // [VARIABLE "my_unique_bucket"]
+  // [VARIABLE "google-cloud-java-tests@java-docs-samples-tests.iam.gserviceaccount.com"]
+  public Acl getBucketAcl(String bucketName, String userEmail) {
+    // [START storagePrintBucketAclForUser]
```
Comment (garrettjonesgoogle) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java]: `storagePrintBucketAclForUser` doesn't seem right, is that what you want?

Comment (jerjou) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java]: Yeah - there's a [nascent effort](go/veneer-docs-samples) to start standardizing on region tag names across languages, and that's what they wanted for this sample. Why? What seems odd about it?

Comment (garrettjonesgoogle) [inline google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java]: Oh it was specifically that it was more different from the snippet name than for other snippets. If it's intentional, then it's fine.

**Pull request merged**